tests/test_affine_builtins.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from xdsl.ir import AffineExpr
+
+
+def test_affine_expr():
+    # d0, s0
+    a = AffineExpr.dimension(0)
+    b = AffineExpr.symbol(0)
+
+    # (5 * d0) + s0 + 1
+    c = (a * 5) + b + 1
+    print(c)
```

Comment:
For testing here, I would `assert` that this is equal to `AffineExpr` constructed manually (without the operators).